pylops/torchoperator.py

```diff
@@ -2,12 +2,12 @@
     "TorchOperator",
 ]
```

Comment:
Actually this is a very special operator, probably the most special of all in PyLops. It is used to bring a PyLops operator into a torch AD chain, so technically speaking I am not really sure it makes any sense for it inherit from PyLops `LinearOperator`. 

Looking at what you did I started to think maybe there is a point if we could chain multiple TorchOperator in the same way we do with PyLops but this doesn't really work because then the new operator has no apply method, which is what we care here...

In short, I believe the most reasonable thing here is to simply remove the inheritance and make it a stand-alone class.

Reply:
Yeah agreed I'll just make the changes and commit..

Reply:
Done already

Reply:
Since the purpose of this class is to provide a PyTorch function, shouldn't it inherit from `torch.autograd.Function` like `_TorchOperator`?